README.md

```diff
@@ -49,6 +55,13 @@ nnoremap <silent> <leader>r :TRun<CR>
 nnoremap <silent> <leader><space> :call TermBufMToggle()<CR>
 ```
 
+You can also replace the first two mappings above with 
+```vim
+:TBuild<CR><C-W><C-W>
+:TRun<CR><C-W><C-W>i
```

Comment:
This looks ugly. Do you think having a separate function maybe called `:TJumpToTerm` would be better? Then this could be used in both the lines shown here to make it more modular and composable. It would also reduce a possible source of bugs.

Reply:
The plugin actually goes to the terminal window when a terminal command is run:

https://github.com/mizlan/termbufm/blob/02fe8bb1720ae16cd17cf384f3bb2e43062d22db/plugin/termbufm.vim#L83-L84

So instead, maybe add an option to **not** go back to the original window? That could be an argument to the function.

Reply:
Also, sending the command without going to the window is something that can probably be fixed sometime.